python/neuroglancer/write_annotations.py

```diff
@@ -24,6 +24,10 @@
 import numpy as np
 import struct
 
+try:
```

Comment:
Can just unconditionally import typing_extensions; on newer versions of Python that is an alias to `typing.Literal`.

Note: module name is misspelled here.

Also need to add `typing_extensions` to install_requires in setup.py